static/js/components/ObservingRunPage.jsx

```diff
@@ -41,9 +97,17 @@ const ObservingRunList = ({ observingRuns }) => {
                 telescopeList,
                 groups
               )}
+              secondary={observingRunInfo(run, instrumentList, telescopeList)}
             />
           </ListItem>
         ))}
+        <input
```

Comment:
I think this should be at the top of the list instead of the bottom. So if the user changes their mind and wants to uncheck they don't have to scroll all the way to the bottom.

Reply:
Good idea!